Description: Correct pod NAME issues in LedgerSMB/DBObject/Asset_Class.pm
 Has an incorrect module name and a comma instead of a dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472550&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Asset_Class.pm
+++ b/LedgerSMB/DBObject/Asset_Class.pm
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-LedgerSMB::DBObject::Asset_Class.pm, LedgerSMB Base Class for Asset Classes
+LedgerSMB::DBObject::Asset_Class - LedgerSMB Base Class for Asset Classes
 
 =head1 SYNOPSIS
 
